test/base-test.js

```diff
@@ -2,15 +2,13 @@ const assert = require("assert");
 const runLintWithFixtures = require("./lib/runLintWithFixtures");
 
 describe("base", () => {
-  it("should get expected errors and warninigs with base config", () => {
-    const result = runLintWithFixtures("base");
+  it("should get expected errors and warninigs with base config", async () => {
+    const result = await runLintWithFixtures("base");
     assert.deepStrictEqual(result, {
       "error.js": {
         errors: [
           "no-var",
           "no-unused-vars",
-          "no-var",
-          "no-redeclare",
           "getter-return",
```

Comment:
The order of the errors is not the same between Node v10 and v12, so I temporarily disable the errors.